pkg/install/1-installresources.go

```diff
@@ -668,9 +664,29 @@ func (i *Installer) installResources(ctx context.Context) error {
 			return err
 		}
 	}
-
 	{
-		ipAddress := lbIP.String()
+		var ipAddress string
+		{
+			// Discover what private IP address was assigned to internal LB
+			lbName := "aro-internal-lb"
+			r, err := azureextended.ParseResourceGroupID(i.doc.OpenShiftCluster.Properties.ClusterProfile.ResourceGroupID)
+			if err != nil {
+				return err
+			}
+			lb, err := i.lbClient.Get(ctx, r.ResourceGroupName, lbName, "")
```

Comment:
use the local `resourceGroup` variable which already exists - you don't need all the azureextended scaffolding in this PR.